README.md

```diff
@@ -69,3 +69,12 @@ HugeGraph relies on the [TinkerPop](http://tinkerpop.apache.org) framework, we r
 Thanks to TinkerPop, thanks to Titan, thanks to DataStax. Thanks to all other organizations or authors who contributed to the project.
 
 You are welcome to contribute to HugeGraph, and we are looking forward to working with you to build an excellent open source community.
+
+## Contact Us
+
+- [Github Issues](https://github.com/apache/incubator-hugegraph/issues): Feedback on usage issues and functional requirements (priority)
+- Feedback Email: [dev@hugegraph.apache.org](dev@hugegraph.apache.org)
+- WeChat public account: Apache HugeGraph, welcome to scan this QR code to follow us.
+
+<img src="https://github.com/apache/incubator-hugegraph-doc/blob/master/assets/images/wechat.png?raw=true" alt="QR png" width="350" />
```

Comment:
I use html tag to reduce the size of the image, other repo could refer this

Reply:
Hi @imbajin ，Is this the picture you were expecting 
![image](https://github.com/apache/incubator-hugegraph/assets/87920097/9f6fa0fe-2d78-4a84-8470-93cb71d0de6a)